scripts/package

```diff
@@ -53,11 +53,13 @@ if [ ! -d build/charts ]; then
     # Extract the tarballs in the list, skip extracting CRD charts
     for TGZ_PATH in $LATEST_TGZ_PATHS; do
         TGZ_REL_PATH=$CHART_REPO_DIR/$TGZ_PATH
+        TGZ_EXTRACT_PATH=$(dirname $CHART_REPO_DIR/${TGZ_PATH##released/})
         if [[ $TGZ_PATH == *crd*.tgz ]]; then
             echo "Skipped CRD: $TGZ_REL_PATH"
         else
-            echo "Extract: $TGZ_REL_PATH"
-            tar -xvf $TGZ_REL_PATH -C $(dirname $TGZ_REL_PATH)
+            echo "Extract: $TGZ_REL_PATH to $TGZ_EXTRACT_PATH"
+            mkdir -p $TGZ_EXTRACT_PATH
+            tar -xvf $TGZ_REL_PATH -C $TGZ_EXTRACT_PATH
```

Comment:
So in short: 
* on release branches tgz's get unfurled in the same dir
* on dev branches a `./assets` dir is made (from `./released/assets`) and  tgz's get unfurled in there

Reply:
Is it just replace the tgz url so that `./released/assets/x.y.z.00.tgz` -> `./assets/x.y.z.00.tgz`?

Reply:
For release branches, everything gets extracted to the `assets` directory.

For dev branches, the charts are in the `released` directory, but get extracted to the `assets` directory.

Reply:
And in both cases the `/assets` dir gets passed to image export.